pwndbg/disasm/color.py

```diff
@@ -57,9 +57,11 @@ def instruction(ins):
         asm = asm.replace(ins.mnemonic, pwndbg.color.bold(ins.mnemonic))
 
     # If we know the conditional is taken, mark it as green.
-    if ins.condition:
+    if ins.condition is True:
         asm = pwndbg.color.green(u'✔ ') + asm
-    else:
+    elif ins.condition is False:
+        asm = pwndbg.color.red(u'✘ ') + asm
+    else: # ins.codition is None
```

Comment:
I'd prefer this check to be explicit for `is None` and then just truthiness (`if ins.condition` vs `... is True`)